### DISCUSSION TITLE
run the main.qt.py the following error occurred.

### DISCUSSION THREAD

**Discussion by ucas-xiang** [IRVLUTD/Proto-CLIP#13] [Q&A] [answered]
![1706063596590](https://github.com/IRVLUTD/Proto-CLIP/assets/156390828/48acfae5-a83b-47f1-a43e-89ac7de0d279)

**Comment by jishnujayakumar** [comment, marked as answer]
Hi @ucas-xiang,

Please check now and let me know if it worked.
